service/src/main/java/org/cbioportal/service/util/AlterationEnrichmentUtil.java

```diff
@@ -62,7 +62,7 @@
             AlterationEnrichment alterationEnrichment = new AlterationEnrichment();
             alterationEnrichment.setEntrezGeneId(gene.getEntrezGeneId());
             alterationEnrichment.setHugoGeneSymbol(gene.getHugoGeneSymbol());
-            alterationEnrichment.setCytoband(gene.getCytoband());
+            //alterationEnrichment.setCytoband(gene.getCytoband());
```

Comment:
can we remove this if it's not going to be used or is there a plan to replace it with the reference_genome_gene.CYTOBAND? If so then can you add a TODO here?